### ISSUE TITLE
[Bug Bash] The splitter in the PM UI cannot be dragged after resizing the VS window from the full-screen mode

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#13421]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

Dev\6.11.0.34

### Worked before?

It is a regression since it doesn’t repro on D16.11\34601.136 and D17.9\34601.282.

### Impact

It bothers me. A fix would be nice

#### Repro Steps: 
1. Create a C# Console App (.NET Core 8.0) project in VS.
2. Right-click the project in Solution Explorer and select "Manage NuGet Packages…" menu item to open PM UI.
3. Resize the whole VS window from the default full-screen mode.
4. Drag the splitter in the PM UI to the left or right.

#### Expected Result: 
The splitter in the PM UI can be dragged to left or right.

#### Actual Result:
The splitter in the PM UI cannot be dragged to left or right as the recording below.
![SplitterCannotBeDragged](https://github.com/NuGet/Home/assets/20632625/2b67b481-d857-42f1-8e3a-bb1c522e3c65)

#### Notes:
1. Repro rate: 100%.
2. It doesn’t repro when the VS window is in full-screen mode by default (at the end of the recording).
3. It also reproes for the splitter of solution-level PM UI.
4. It also reproes for the 'Installed'/'Updates'/'Consolidate' tab.

**Comment by donnie-msft** [comment]
I repro'd this. ~I believe it's a regression on my change here~. Looking into it.

**Comment by donnie-msft** [comment]
I was able to repro this issue on VisualStudio.17.Release/17.9.6+34728.123


https://github.com/NuGet/Home/assets/49205731/15c67466-5ec6-4c53-b1c2-307cead09f10

**Comment by kartheekp-ms** [comment]
> It is a regression since it doesn’t repro on D16.11\34601.136 and D17.9\34601.282.

Given that @v-luzh mentioned that this issue doesn't repro on the above VS versions, I added `RegressionFromPreviousRTM` label.

**Comment by kartheekp-ms** [comment]
I agree with @donnie-msft. I am able to reproduce this issue in version 17.9.6. One behavior I noticed is that if you resize the window slightly, you can drag the splitter. However, when I resized the window even further, I found that I was unable to drag the splitter.

![pmui-splitter](https://github.com/NuGet/Home/assets/52756182/a64d91d6-6983-48e9-88db-b5aca2ba4a29)

**Comment by donnie-msft** [comment]
@v-luzh Thank you for your attention to looking for nuanced UI bugs like this. In this particular case, I believe this is not a bug. When resizing the PM UI doc window, notice that a horizontal scrollbar appears. When the content doesn't fit horizontally between the 2 panels, then scrolling allows the customer to scroll to see the desired content.

If there's customer input on wanting the ability to slide the splitter despite the very small amount of horizontal width, I can look into what the work will be to modify our WPF controls to be very adaptive.

Otherwise, consider closing this for now. Thanks again!

**Comment by v-luzh** [comment]
Hi @donnie-msft, I see. Make sense for me, please discuss with others in the triage meeting and make a decision, thanks.

**Comment by SueSu01** [comment]
It still reproes on VS Main\35222.175 + NuGet Client Dev\6.12.0.82.

**Comment by v-luzh** [comment]
It is fixed on VS Main\12206.68+NuGet Client Dev\7.12.0.4.
